integration/cluster.go

```diff
@@ -80,7 +88,7 @@ func (c *cluster) fillClusterForMembers() error {
 	addrs := make([]string, 0)
 	for _, m := range c.Members {
 		scheme := "http"
-		if !m.PeerTLSInfo.Empty() {
+		if m.PeerTLSInfo != nil {
```

Comment:
hrm... OK. I think in the real etcdserver codebase, tls is always created. So nil case should not exist. But I need to double check. It is fine here though.